pkg/standalone/common.go

```diff
@@ -0,0 +1,20 @@
+package standalone
+
+import (
+	"os"
+	path_filepath "path/filepath"
+	"runtime"
+)
+
+// getDefaultComponentsFolder returns the hidden .components folder created under install directory at init time
+func getDefaultComponentsFolder() string {
+	const daprDirName = ".dapr"
+	const componentsDirName = "components"
+	daprDirPath := os.Getenv("HOME")
+	if runtime.GOOS == daprWindowsOS {
+		daprDirPath = path_filepath.Join(os.Getenv("HOMEDRIVE"), os.Getenv("HOMEPATH"))
```

Comment:
The user profile env var should work here.

Reply:
fixed